client/src/components/Card/Card.css

```
@@ -0,0 +1,73 @@
+.card-container {
+	display: flex;
+	flex-direction: column;
+	align-items: center;
+	justify-content: center;
+	width: 350px;
+	margin: 0 0 2rem 0;
+	position: relative;
+	border-radius: 25px;
+	/* just for test for now */
+	border: 1px solid var(--black);
+	box-shadow: rgba(0, 0, 0, 0.16) 0px 3px 6px, rgba(0, 0, 0, 0.23) 0px 3px 6px;
+	transition: all 0.2s ease-in-out;
+}
+
+.card-container:hover {
+	/* just for test */
+	box-shadow: rgba(0, 0, 0, 0.05) 0px 0px 0px 1px;
+}
+
+.card-image-container {
+	/* changed this to make border visible */
+	width: 348px;
+	height: 250px;
+	overflow: hidden;
+	margin-bottom: 0.5rem;
+	border-radius: 25px;
+}
+
+.card-image {
+	width: 100%;
+	height: 100%;
+	object-fit: cover;
+}
+
+.card-subcontainer {
+	display: flex;
+	flex-direction: column;
+	justify-content: center;
+	align-items: center;
+	width: 70%;
+	gap: 0.5rem;
+}
```

This is just a suggestion and goes with the below suggestion, try it and see if you like it
As you said you will change this later with a specific card styling.

```suggestion
.card-subcontainer {
	position: relative;
	margin-bottom: 1.5rem;
	display: flex;
	flex-direction: column;
	justify-content: center;
	align-items: center;
	width: 75%;
	gap: 0.5rem;
}
```